src/python/pants/backend/python/rules/coverage_integration_test.py

```diff
@@ -97,16 +96,12 @@ def test_arithmetic():
                       name="multiply",
                       sources=["test_multiply.py"],
                       dependencies=['{tmpdir_relative}/src/python/project'],
-                      coverage=['project.lib'],
```

Comment:
This is a main motivation for this PR. Previously, if the user did not specify `coverage`, then they wouldn't get any data for `project.lib`. Now, they get it automatically. The user will also automatically get data on this test itself.